.github/workflows/docs/check-build-docs

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
This script is actually failing (but we are swallowing the error from `build-docs`: see the logs.

Reply:
Repaired

Reply:
Thank you for spotting!